frontend/components/TableContainer/TableContainer.tsx

```diff
@@ -152,7 +153,7 @@ const TableContainer = ({
   const [sortDirection, setSortDirection] = useState(
     defaultSortDirection || ""
   );
-  const [pageIndex, setPageIndex] = useState(DEFAULT_PAGE_INDEX);
+  const [pageIndex, setPageIndex] = useState(defaultPageIndex);
```

Comment:
Cleaned up how pageIndex is being set in table container to ensure correct default page (which might be page 5 if someone uses a url that sets the `page=5`)

Reply:
Already included in other PR that's already merged